### ISSUE TITLE
[Bug]: --include_all_ctgs: expected type: String, found: Boolean (true)

### ISSUE THREAD

**Issue by felixm3** [epi2me-labs/wf-human-variation#5]
### What happened?

Trying to call variants on a whole genome sample with this command

```

wgs_hg38_bed=~/tools/bedtools/hg38.wgs.50k.bed
hg38_ref=~/tools/hg38/hg38.fa

nextflow run epi2me-labs/wf-human-variation \
    -w output/workspace \
    -profile conda \
    --snp --sv \
    --bam $1 \
    --bed $wgs_hg38_bed \
    --ref $hg38_ref \
    --model /home/felixm3/.conda/envs/clair3a/bin/models/r941_prom_sup_g5014 \
    --out_dir output/$start \
    --threads 56 \
    --include_all_ctgs

```

I get the below error

```
N E X T F L O W  ~  version 21.10.6
Launching `epi2me-labs/wf-human-variation` [modest_euler] - revision: 4497a3b1ad [master]

ERROR: Validation of pipeline parameters failed!


* --include_all_ctgs: expected type: String, found: Boolean (true)
```

Am I using `--include_all_ctgs` incorrectly? If so, what's the right syntax?

### Operating System

macOS

### Workflow Execution

Command line

### Workflow Execution - Execution Profile

Conda

### Workflow Version

No idea. Where do I find this information?

### Relevant log output

```shell
N E X T F L O W  ~  version 21.10.6
Launching `epi2me-labs/wf-human-variation` [modest_euler] - revision: 4497a3b1ad [master]

ERROR: Validation of pipeline parameters failed!


* --include_all_ctgs: expected type: String, found: Boolean (true)
```

**Comment by SamStudio8** [comment]
Hi @felixm3, thanks for reporting this. We are seeing something similar with one of the other options. I'll fix these discrepancies in our next release. In the meantime you should be able to use `--include_all_ctgs y`.

**Comment by SamStudio8** [comment]
Broken booleans should now be fixed in the 0.2.0 release.

**Comment by felixm3** [comment]
Confirmed that this now works as expected.

Thanks! 😀

```
nextflow run epi2me-labs/wf-human-variation \
        -w output/workspace \
        -profile conda \
        --snp --sv \
        --bam $1 \
        --bed $wgs_hg38_bed \
        --ref $hg38_ref \
        --model /clair3a/bin/models/r941_prom_sup_g5014 \
        --out_dir output/$start \
        --threads 8 \
        --include_all_ctgs
```
